### DISCUSSION TITLE
Dark Mode type skins are not applied.

### DISCUSSION THREAD

**Discussion by yuqwe** [helge17/tuxguitar#307] [General]
The background and UI are not darkened after setting skins such as Oxygen Dark, Symbolic Dark, etc.

I have tried reinstalling, deleting folders, applying skins from other developers, changing the OS basic theme, etc. but no improvement.
Even in past versions (1.5.3), the UI did not darken, even with other Dark themes.
The PC is Windows 10 (Pro 22H2) and I have never done anything to modify the OS theme or registry.
If there is a possible cause, I would like to know.

I searched for the same phenomenon but could not find it, so I asked my first question on github.
I am Japanese, sorry if my English is hard to read or if I am not using this place correctly.
Thanks very much for taking a look.
![capture1283202432494021](https://github.com/helge17/tuxguitar/assets/78135052/444b9d32-fee2-49e9-af92-f463864b754f)

**Comment by guiv42** [comment]
Hi,

the description of your issue is very clear. Unfortunately, this is an already known limitation, for which we could not find a solution yet. See issue #27.

**Reply by yuqwe** [reply]
Thank you for your quick reply...!
My search was shallow and I apologize for the inconvenience.
I had no idea that a solution had not been found! I hope you can find the cause of the problem.
I plan to use it as is for a while.

**Reply by guiv42** [reply]
No need to apologize, it's always interesting to have user feedback ;)

**Comment by guiv42** [comment]
closing discussion, issue tracked by #27
